### ISSUE TITLE
Potential issues with releasing iOS app w/ Themis on M1

### ISSUE THREAD

**Issue by vixentael** [cossacklabs/themis#863]
@deszip brings us news – he is trying to build project with Themis on "normal laptop" and it's working. However, building the same project on M1 using macOS beta2 leads to linking errors: Xcode can't find `armv7` architecture slice.

It's unclear whether it's Themis/OpenSSL building problem OR M1/macOS beta related problems. Need to investigate.

**Environment:**
- OS: latest iOS
- Hardware: Apple M1, macOS beta2
- Themis version: 0.13.10
- Installation way: cocoapods

xcode 13
![IMAGE 2021-07-16 15:04:22](https://user-images.githubusercontent.com/2877920/125944646-0720dbc6-5de6-4734-ab2f-b6ed46663265.jpg)

xcode 12
![IMAGE 2021-07-16 15:04:24](https://user-images.githubusercontent.com/2877920/125944671-cb461521-934b-472b-bb6d-8a19f0e89f7e.jpg)

**Comment by vixentael** [comment]
@rad this is something for us to check with upcoming release:
- cocoapods
- M1 laptop
- Build, Debug, Release, submit to appstore

We should verify that Themis+cocoapods apps work without issues on M1 during the whole project lifecycle.

**Comment by radetsky** [comment]
Today I tested the issue. 

Model Name:	MacBook Pro
Chip:	Apple M1 Pro
Memory:	16 GB
System Firmware Version:	7459.141.1
macOS: 12.5.1
XCode: 13.4.1 
Cocoapods: 1.11.3 installed from homebrew

Steps which I have done: 
1. Make the empty project 'Application for iOS' with SwiftUI 
2. Run 'pod init', add pod 'themis' to the Podfile and run 'pod install'
3. Add 'import themis' and code to the ContentView.swift file to render symmetric key, asymmetric key pair, and encrypted example text 'Hello, Themis!' without context 
4. I saw the preview 
5. Run with debug and release modes on the simulator 
6. Archive it and upload it to App Store 
7. Run internal testing and download binary through the TestFlight app
8. Test application natively on the iPhone X

Test project you find at https://github.com/radetsky/ThemisTestSwiftUIPods 

![IMG_1826](https://user-images.githubusercontent.com/311543/189658464-795b16d6-6845-4442-87b0-ec980fe27a3d.PNG)
